content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,203 @@
+
+- :guilabel:`Confirmed order`: the document is sent to customers upon the confirmation of an order.
+  This is best for user manuals and other supplemental documents.
+
+- :guilabel:`Inside quote`: the document is included in the PDF of the quotation, between the header
+  pages and the quote table.
+
+.. example::
+   How the document link appears inside a quotation.
+
+    .. image:: pdf_quote_builder/pdf-inside-quote-sample.png
+       :align: center
+       :alt: Sample of an uploaded pdf with the inside quote option chosen in Odoo Sales.
+
+Lastly, in the :guilabel:`E-Commerce` section, decide whether or not to :guilabel:`Show on product
+page` on the front-end (in the online store).
+
+.. example::
+   How the document link appears on a typical product page in the online store.
+
+    .. image:: pdf_quote_builder/show-product-page.png
+       :align: center
+       :alt: Showing a link to an uploaded document on a product page using Odoo Sales.
+
+Dyanmic text values (product-specific)
+--------------------------------------
+
+While creating custom PDFs for quotes, using any standard PDF editor software, certain dynamic texts
+should be utilized throughout the design of the PDF to ensure Odoo can automatically fill in the
+appropriate content accordingly.
+
+Dynamic text values are, essentially, coded placeholders that Odoo fills in automatically, according
+to the specific information related to the quote.
+
+Below are the dynamic text values and what they will represent in the custom PDFs:
+
+- :guilabel:`name`: Order Reference
+- :guilabel:`partner_id_name`: Customer Name
+- :guilabel:`user_id_name`: Salesperson Name
+- :guilabel:`amount_untaxed`: Untaxed Amount
+- :guilabel:`amount_total`: Total Amount
+- :guilabel:`delivery_date`: Delivery Date
+- :guilabel:`validity_date`: Expiration Date
+- :guilabel:`client_order_ref`: Customer Reference
+
+Product-specific dynamic text values are as follows:
+
+- :guilabel:`description`: Product Description
+- :guilabel:`quantity`: Quantity
+- :guilabel:`uom`: Unit of Measure (UoM)
+- :guilabel:`price_unit`: Price Unit
+- :guilabel:`discount`: Discount
+- :guilabel:`product_sale_price`: Product List Price
+- :guilabel:`taxes`: Taxes name joined by a comma (`,`)
+- :guilabel:`tax_excl_price`: Tax Excluded Price
+- :guilabel:`tax_incl_price`: Tax Included Price
+
+PDF quote
+=========
+
+When a quotation with a configured PDF-related quote has been confirmed, click the :guilabel:`⚙️
+(gear)` icon to reveal a drop-down menu. From this drop-down menu, select :guilabel:`Print`, then
+select :guilabel:`PDF Quote`.
+
+.. image:: pdf_quote_builder/drop-down-print-pdf.png
+   :align: center
+   :alt: Print pdf quote option on drop-down menu located on confirmed sales order in Odoo Sales.
+
+Doing so instantly downloads the PDF quote. When opened, the PDF quote, along with the configured
+product PDF that was set to be visible inside the quote, can be seen.
```

Comment:
I'm still a little unsure why it's useful to see the quote. Is it to confirm that the dynamic placeholders work as intended?